### ISSUE TITLE
Pytest RemovedInPytest4Warning Warning

### ISSUE THREAD

**Issue by orenhecht** [manahl/pytest-plugins#100]
Getting the following warning:

```
/usr/local/lib/python3.6/dist-packages/pytest_verbose_parametrize.py:20
  /usr/local/lib/python3.6/dist-packages/pytest_verbose_parametrize.py:20: RemovedInPytest4Warning: MarkInfo objects are deprecated as they contain merged marks which are hard to
 deal with correctly.
  Please use node.get_closest_marker(name) or node.iter_markers(name).
  Docs: https://docs.pytest.org/en/latest/mark.html#updating-code
    if 'ids' not in markers.kwargs:
```

**Comment by jcfr** [comment]
@eeaston This issue can now be closed. It was fixed in https://github.com/jcfr/pytest-plugins/commit/abffd6727ed56fba3b9154a6254d29191d8eba4b#diff-608c200a11232b8a1fe71095f515b966

**Comment by eeaston** [comment]
Released in 1.7.0 - many apologies all for the delay in releasing to pypi.
